File: VBNetCore.Controls/InputTextBox.cs
using System;
using System.Text.RegularExpressions;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Input;
using System.Windows.Media;

namespace NetCore.Controls
{
    public class InputTextBox : TextBox
    {
        static InputTextBox()
        {
            DefaultStyleKeyProperty.OverrideMetadata(typeof(InputTextBox), new FrameworkPropertyMetadata(typeof(InputTextBox)));
        }
    }
}

File: VBNetCore.Controls/InputTextBox.vb
Imports System.Windows
Imports System.Windows.Controls

Namespace VBNetCore.Controls
    Public Class InputTextBox
        Inherits TextBox
        Shared Sub New()
            Call DefaultStyleKeyProperty.OverrideMetadata(GetType(InputTextBox), New FrameworkPropertyMetadata(GetType(InputTextBox)))
        End Sub
    End Class
End Namespace
